src/gui/search/searchjobwidget.cpp

```diff
@@ -208,7 +217,15 @@ void SearchJobWidget::cancelSearch()
 
 void SearchJobWidget::downloadTorrents(const AddTorrentOption option)
 {
-    const QModelIndexList rows {m_ui->resultsBrowser->selectionModel()->selectedRows()};
+    if (!BitTorrent::Session::instance()->isRestored())
+    {
+        QMessageBox::warning(this, tr("BitTorrent Session isn't restored")
+                             , tr("BitTorrent Session isn't restored yet. Adding new torrents is impossible. Please wait...")
```

Comment:
>Adding new torrents is impossible.

It should mention when it is possible, i.e. "Adding new torrents is prohibited until session is restored."

This applies to other similar messages.